.github/workflows/ci.bazelrc

```diff
@@ -5,3 +5,4 @@ test --test_output=errors
 build --disk_cache=~/.cache/bazel
 build --repository_cache=~/.cache/bazel-repo
 test --test_env=XDG_CACHE_HOME
+build --noworker_sandboxing
```

Comment:
does this suggest that users are required to add this flag as well?

Reply:
not really just wanted to know if this was causing module resolution errors. I'll remove this since linker does not work with worker_sandboxing